llvm/lib/Target/Mips/MipsCCState.h

```diff
@@ -191,6 +192,16 @@ class MipsCCState : public CCState {
     return Return;
   }
 
+  bool CheckCallReturn(const SmallVectorImpl<ISD::OutputArg> &ArgsFlags,
+                   CCAssignFn Fn, const Type *RetTy) {
+    PreAnalyzeCallReturnForF128(ArgsFlags, RetTy);
+    PreAnalyzeReturnForVectorFloat(ArgsFlags);
```

Comment:
I don't understand how this fixes it, but this looks like hacking around something that shouldn't require special casing

Reply:
Add a new function to check whether call funtion return type was origin from fp128.